scripts/tf_cnn_benchmarks/models/resnet_model.py

```diff
@@ -77,6 +77,48 @@ def bottleneck_block_v1(cnn, depth, depth_bottleneck, stride):
     cnn.top_layer = output
     cnn.top_size = depth
 
+def bottleneck_block_v15(cnn, depth, depth_bottleneck, stride):
+  """Bottleneck block with identity short-cut for ResNet v1 with 3x3 stride 2.
+
+  ResNet v1.5 is the informal name for ResNet v1 where the first 3x3 conv in
+  each block has a stride 2 an the 1x1 a stride 1.  ResNet v1 has the stride 2
```

Comment:
Replace "has a stride 2" with "has stride 2". Replace "an" with "and". Replace "a stride 1" with "has stride 1".

Reply:
I reworked the description completely.  Let me know what you think.